python/py-sentencepiece/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        google sentencepiece 0.1.84 v
+revision            0
+name                py-${github.project}
+categories          python textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python wrapper for ${github.project}
+
+long_description    SentencePiece is an unsupervised text tokenizer\
+                    and detokenizer mainly for Neural Network-based\
+                    text generation systems where the vocabulary size\
+                    is predetermined prior to the neural model\
+                    training. SentencePiece implements subword units\
+                    (e.g., byte-pair-encoding (BPE) \[Sennrich et al.\])\
+                    and unigram language model \[Kudo.\]) with the\
+                    extension of direct training from raw\
+                    sentences. SentencePiece allows us to make a\
+                    purely end-to-end system that does not depend on\
+                    language-specific pre/postprocessing.
+
+homepage            https://github.com/google/sentencepiece
+
+checksums           rmd160  91bc4a0a23d9a4daf98abbb9ec83aed2241fdc42 \
+                    sha256  dacab5fe63875b94538e06f4995ffb8cf35400708f3c333263f7b72e2e5b43de \
+                    size    11828855
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    # see https://github.com/google/sentencepiece/blob/master/python/setup.py
```

Comment:
but you are missing ```pkgconfig``` as build dependency; you can see it wants to call that in the setup.py file and if you try to build it in trace-mode (which in the PR-checklist and you say you did) it will fail.

Reply:
Thanks done.

Reply:
not done correctly - I said a dependency on ```pkgconfig``` not ```py-pkgconfig``` ;) 

I am happy to review things, but you'll also have to be a bit more careful yourself and look at the projects for which you're submitting ports. If you just check the ```setup.py``` file and you'll see it right there... The only reason that it does work now in tracemode (which you tried, correct?) is because ```pkgconfig``` is a dependency of ```py-pkgconfig```, but it's not the correct thing to do.

Reply:
Done thanks.